modules/web/js/ballerina/components/transform-render.js

```diff
@@ -968,7 +968,7 @@ addComplexParameter(parentId, struct) {
         const xTargetPointer = 0;
         let yTargetPointer = 0;
         const functionGap = 30;
-        const svgLines = $('#' + self.placeHolderName + '> svg');
+        const svgLines = $('#' + self.viewId + '> svg');
```

Comment:
We have used a different solution. The viewId in this case should differentiate svgLines in separate tabs. So $ is fine here. :)